frontend/test/__support__/ui.tsx

```diff
@@ -61,7 +61,8 @@ export function renderWithProviders(
   }
 
   const history = withRouter
-    ? createMemoryHistory({ entries: [initialRoute] })
+    ? // eslint-disable-next-line react-hooks/rules-of-hooks
+      useRouterHistory(createMemoryHistory)({ entries: [initialRoute] })
```

Comment:
Without this `props.location.query` would be undefined and fail the test.

https://github.com/metabase/metabase/blob/30317c6b281fb601db9ae5372877150d2784afc6/frontend/src/metabase/admin/databases/containers/DatabaseListApp/DatabaseListApp.jsx#L43

This change matches the app and unit test's routing behavior.